src/main/python/deep_qa/training/text_trainer.py

```diff
@@ -94,9 +93,9 @@ def __init__(self, params: Dict[str, Any]):
         # don't want to set them now, because they use max length information that only gets set
         # after reading the training data.
         self.embedding_layers = {}
-        self.sentence_encoder_layer = None
-        self.word_encoder_layer = None
-        self.seq2seq_encoder_layer = None
+        self.sentence_encoder_layers = {}
+        self.word_encoder_layers = {}
```

Comment:
I would remove this field entirely, and change `self.sentence_encoder_layers` to `self.encoder_layers`.